modules/puppetca_token_whitelisting/puppetca_token_whitelisting_autosigner.rb

```diff
@@ -0,0 +1,96 @@
+require 'jwt'
+require 'openssl'
+
+module ::Proxy::PuppetCa::TokenWhitelisting
+  class Autosigner
+    include ::Proxy::Log
+    include ::Proxy::Util
+
+    def sign_all
+      Proxy::PuppetCa::TokenWhitelisting::Plugin.settings.sign_all
```

Comment:
I am still wondering if it should allow reading a whitelist from the actual autosign file but this might be enough.

Reply:
I think falling back to the `autosign.conf` could be confusing. Allowing this for people who really really want to put the `*` in the file should be fine.